README.md

```diff
@@ -153,10 +158,17 @@ repository.
 
 ### Packaging Configuration Files
 
-In the final packages produced by an ezbake build, there will be a "config' directory
+By default, in the final packages produced by an ezbake build, there will be a "config" directory
 (usually `/etc/puppetlabs/<ezbake-project-name>`) that contains all of the final
 configuration files for the ezbake application.
 
+If the `:bootstrap-source` setting is set to `:services-d`, there will also be an additional
+configuration directory under the installation directory:
+`/opt/puppetlabs/server/apps/{project-name}/config/`
+This directory will then be used to hold a `services.d/` directory that contains bootstrap files.
+Usually this will be for bootstrap entries the application could not run without, and which should
+not be modified by the user. See [Bootstrap Files](#bootstrap_files)
```

Comment:
I think I might add a subsection markdown header before this paragraph, called something like "bootstrap-source".  Then explicitly mention the two legal values for the enum here.  It's fine to keep this paragraph brief and defer the more detailed explanation to the later section that you added, but I think you should at least call out the legal values for the setting.